src/clean-config.js

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+const PhishingDetector = require('./detector')
+const config = require('./config.json')
+
+const SECTION_KEYS = {
+  blocklist: 'blacklist',
+  allowlist: 'whitelist',
+};
+
+const exitWithUsage = (exitCode) => {
+  console.error(`Usage: ${
+    process.argv.slice(0,2).join(' ')
+  } ${
+    Object.keys(SECTION_KEYS).join('|')
+  }`);
+  console.error('Removes redundant entries from config section and writes filtered config to standard output');
+  process.exit(exitCode);
+};
+
+const listName = process.argv[2];
+const section = SECTION_KEYS[listName];
+if (!section) {
+  exitWithUsage(1);
+}
+
+const resultFilter = listName === 'blocklist'
```

Comment:
(should) A more descriptive name would help understand the logic below.

`isResultingBehaviorExpectedForCurrentList`

(could) Also, while redundant, passing the listName as argument would make it easier still to understand what this function is doing without scrolling up to it while reading the for loop body.